### DISCUSSION TITLE
PHP 8.1 bcmath missing

### DISCUSSION THREAD

**Discussion by zofe** [oerdnj/deb.sury.org#1910] [Q&A] [answered]
apt install php8.1-bcmath

https://packages.sury.org/php/pool/main/p/php8.1/php8.1-bcmath_8.1.13-1+0~20221126.29+debian10~1.gbpfee7cc_amd64.deb  404  Not Found [IP: 84.17.59.115 443]

**Comment by oerdnj** [comment, marked as answer]
Yes, that's correct.

**Comment by zofe** [comment]
there is an alternative source to install php8.1-bcmath on my debian 10?
sorry for the question 
thanks

**Reply by oerdnj** [reply]
You ignored the [issue templates](https://github.com/oerdnj/deb.sury.org/issues/new/choose) and filled an issue with single line only. You also obviously haven't even tried to resolve the issue yourself, so what exactly do you expect?

**Reply by zofe** [reply]
sorry, I will be more respectful in the future, an apt-get update  before fixed it.
